src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -54,7 +54,7 @@ public class EditCommand extends Command {
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Client: %1$s";
```

Comment:
```suggestion
    public static final String MESSAGE_EDIT_CLIENT_SUCCESS = "Edited Client: %1$s";
```

Since the subject has been changed from 'Person' to 'Client', would it be more intuitive to change the String name as well?